src/gameover.cpp

```diff
@@ -57,10 +57,12 @@ void Gameover::update(float frameTime) {
 	if (input->wasKeyPressed(ENTER_KEY)) {
 		if (sceneData.scores.p1Score != sceneData.scores.p2Score) {
 			// Remove enter char
-			std::string name = input->getTextIn().substr(0, input->getTextIn().size() - 1);
+			std::string name = input->getTextIn().substr(0, input->getTextIn().size());
```

Comment:
No need `substr` since `enter` WM_CHAR is not appended to input's `textIn` string.